### PR TITLE
Added struct for gas detection module

### PR DESCRIPTION
**Pull request by crisvdn**

### CODE REVIEW

File: code/headers/frame_types.hpp [hunk 1]
```diff
@@ -514,6 +515,21 @@ namespace r2d2 {
         // explanation for each status.
         uint16_t status;
     };
+
+    /* 
+    * This frame will be send from the gas detection module. It will send the gas_id(which corresponds to a specific gas)
```
Comment (LRstudentHU) [inline code/headers/frame_types.hpp]: Could you place a newline after 'module.'? Otherwise it's a bit long.

File: code/headers/frame_types.hpp [hunk 2]
```diff
@@ -514,6 +515,21 @@ namespace r2d2 {
...
     };
+
+    /* 
+    * This frame will be send from the gas detection module. It will send the gas_id(which corresponds to a specific gas) 
+    * and the gas_value will be the value of gas in parts per million.
+    * Refer to the wiki for more information:
+    * wiki page: https://github.com/R2D2-2019/R2D2-2019/wiki/Gas-Detection
+    */
+    R2D2_PACK_STRUCT
+    struct frame_gas_s{
```
Comment (LRstudentHU) [inline code/headers/frame_types.hpp]: Please add a space before `{`

File: code/headers/frame_types.hpp [hunk 3]
```diff
@@ -514,6 +515,21 @@ namespace r2d2 {
...
+        // The gas value in parts per million.
+        uint16_t gas_value;
+        // The gas id which corresponds to a specific gas.
+        // For example: 0 is LPG, 1 is Co, 2 is smoke.
+        uint8_t gas_id;
```
Comment (LRstudentHU) [inline code/headers/frame_types.hpp:532]: We have a `frame_enums.hpp`, where you could create an `enum` for the differect gas types. That would make this clearer.

**Review by LRstudentHU — CHANGES_REQUESTED** [review]

File: code/headers/frame_types.hpp (hunk 1 above)
Comment (crisvdn) [reply code/headers/frame_types.hpp]: Done

File: code/headers/frame_types.hpp (hunk 2 above)
Comment (crisvdn) [reply code/headers/frame_types.hpp]: Done

File: code/headers/frame_types.hpp (hunk 3 above)
Comment (crisvdn) [reply code/headers/frame_types.hpp:532]: Gas type enum added, please check if this is alright

**Review by LRstudentHU — APPROVED** [review]

**Pull request merged**